packages/kit/src/core/adapt/builder.js

```diff
@@ -144,6 +145,13 @@ export function create_builder({
 			}
 		},
 
+		findServerAssets(route_data) {
```

Comment:
should this have a jsdoc since it's public? there's one added on `find_server_assets`, but people might not be able to easily see that one

Reply:
it has one, the `Builder` interface is described in `public.d.ts` https://github.com/sveltejs/kit/pull/11649/files#diff-a174e22e6d675073a963705b97687d42e219c2e05eb1cd6d5811c2581d416a8eR110-R113